HelpSource/Classes/Pfunc.schelp

```diff
@@ -4,7 +4,7 @@ categories:: Streams-Patterns-Events>Patterns>Function
 related:: Classes/Pfuncn, Classes/FuncStream
 
 description::
-Returns a link::Classes/Stream:: that returns values from the code::nextFunc::.
+Returns a link::Classes/Stream:: that returns values from the code::nextFunc::. To return a pattern, see Plazy.
```

Comment:
should `Plazy` not be a link? link::Classes/Plazy

Reply:
Not too sure on how GitHub works, but I added another commit correcting this shortly after. It should be visible in the second commit. Is there something I have to do?

Reply:
I see it - all good!